traits/has_traits.py

```diff
@@ -1358,8 +1358,14 @@ def trait_get(self, *names, **metadata):
 
         Parameters
         ----------
-        names : list of strings
-            A list of trait attribute names whose values are requested.
+        names : optional
```

Comment:
Let's follow the [NumPyDoc style](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) here and use `*names` and `**metadata`.
```suggestion
        *names
```